### PR TITLE
User lesson

### PR DESCRIPTION
**Pull request by NaokiIwamoto**
## Description / Purpose (checkpoints)
User can take "lesson"

Fixes issue: 
<link-to-your-spreadsheet> 
https://docs.google.com/spreadsheets/d/1KanXdYJuprUERp6mQZAxVSb1ssrIL67G0Akeesurc4E/edit?usp=sharing
<task-number>
9

## How To Test This PR
 1. Go "Home page".
 2. Click lesson button "beginner" "Intermediate" or "advance".
 3. Click answer button.
 4. Display the result.
 
 **Commands**
php artisan migrate
Create Category and Word before take lesson.

 **Screenshots**
<Home page>
[Screenshot from Gyazo](https://gyazo.com/73696e9380f8c4a1ac32e6564fae743c)

<Lesson page>
[Screenshot from Gyazo](https://gyazo.com/176121736ea16c8c4b7b1b1e1c8e0955)

<Result page>
[Screenshot from Gyazo](https://gyazo.com/b013e0b57474f99d45751c66b6a5f3d1)

### CODE REVIEW

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -18,7 +18,12 @@
     <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet" type="text/css">
 
     <!-- Styles -->
+<<<<<<< Updated upstream
```
Comment (AubreyV) [inline resources/views/layouts/app.blade.php]: Please do not push conflicts

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -18,7 +18,12 @@
...
     <link href="/css/app.css" rel="stylesheet">
+=======
```
Comment (AubreyV) [inline resources/views/layouts/app.blade.php]: Please do not push conflicts

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -18,7 +18,12 @@
...
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+>>>>>>> Stashed changes
```
Comment (AubreyV) [inline resources/views/layouts/app.blade.php]: Please do not push conflicts

**Review by hayato0309 — APPROVED** [review]
Approved PR#9
1. User can take lessons.
2. User can the result after taking lessons.

**Pull request merged**